src/pages/[platform]/build-a-backend/auth/password-management/index.mdx

```diff
@@ -61,10 +61,450 @@ export const auth = defineAuth({
+
+- [Add social provider sign-in](/[platform]/build-a-backend/auth/add-social-provider/)
+- [Multi-factor authentication](/[platform]/build-a-backend/auth/manage-mfa/)
+
+</InlineFilter>
+<InlineFilter filters={['flutter']}>
+## Reset password
+
+In order to reset your password, use the resetPassword api - this will send a code to the user attribute configured to receive such a reset code (e.g. email or SMS):
+
+```dart
+Future<void> resetPassword(String username) async {
+  try {
+    final result = await Amplify.Auth.resetPassword(
+      username: username,
+    );
+    await _handleResetPasswordResult(result);
+  } on AuthException catch (e) {
+    safePrint('Error resetting password: ${e.message}');
+  }
+}
+```
+
+```dart
+Future<void> _handleResetPasswordResult(ResetPasswordResult result) async {
+  switch (result.nextStep.updateStep) {
+    case AuthResetPasswordStep.confirmResetPasswordWithCode:
+      final codeDeliveryDetails = result.nextStep.codeDeliveryDetails!;
+      _handleCodeDelivery(codeDeliveryDetails);
+      break;
+    case AuthResetPasswordStep.done:
+      safePrint('Successfully reset password');
+      break;
+  }
+}
+```
+
+To complete the password reset process, invoke the confirmResetPassword api with the code you were sent and the new password you want.
+
+```dart
+Future<void> confirmResetPassword({
+  required String username,
+  required String newPassword,
+  required String confirmationCode,
+}) async {
+  try {
+    final result = await Amplify.Auth.confirmResetPassword(
+      username: username,
+      newPassword: newPassword,
+      confirmationCode: confirmationCode,
+    );
+    safePrint('Password reset complete: ${result.isPasswordReset}');
+  } on AuthException catch (e) {
+    safePrint('Error resetting password: ${e.message}');
+  }
+}
+```
+
+## Change password
+
+A signed in user can update their password using the updatePassword api:
+
+```dart
+Future<void> updatePassword({
+  required String oldPassword,
+  required String newPassword,
+}) async {
+  try {
+    await Amplify.Auth.updatePassword(
+      oldPassword: oldPassword,
+      newPassword: newPassword,
+    );
+  } on AuthException catch (e) {
+    safePrint('Error updating password: ${e.message}');
+  }
+}
+```
+## Conclusion
+
+Congratulations! You finished the **Set up user password change and recovery** guide. In this guide, you learned how to enable password changes by your users and help them recover their account and replace their password using a registered email or phone number.
+
+### Next steps
+
+Now that you enabled password management you may also want to add some additional features. We recommend you learn more about:
+
+- [Add social provider sign-in](/[platform]/build-a-backend/auth/add-social-provider/)
+- [Multi-factor authentication](/[platform]/build-a-backend/auth/manage-mfa/)
+</InlineFilter>
+<InlineFilter filters={['android']}>
 ## Reset Password
```

Comment:
know this might be a painful ask now but I think it'll make maintaining it easier. All the headings seem to be the same. Can we group the filters together under each heading? Example

## Reset Password

<Filter for Android/>
<Filter for Swift/>
<Filter for Flutter/>

## Change Password

<Filter for Android/>
<Filter for Swift/>
<Filter for Flutter/>

....

Reply:
Done.